tests/functional/testReindex.js

```diff
@@ -0,0 +1,64 @@
+const assert = require('assert');
+
+const UtapiReindex = require('../../lib/UtapiReindex');
+const redisClient = require('../../utils/redisClient');
+const log = require('../utils/mock/log');
+
+describe('UtapiReindex', () => {
+    let reindex;
+    let redis;
+
+    beforeEach(done => {
+        reindex = new UtapiReindex();
+        redis = redisClient({}, log)
+            .on('ready', done)
+            .on('error', done);
+    });
+
+    afterEach(done => {
+        redis.flushdb(done);
```

Comment:
maybe a good idea to close the client as well if we're re-instantiating a redisclient for every test

Reply:
Good idea, I will update that.